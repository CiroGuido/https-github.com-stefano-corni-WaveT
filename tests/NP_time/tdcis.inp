&general 
n_ci=10
n_ci_read=15
mol_cc= 0. 0. 0. 
medium='nan'
n_f=1
dt=0.20
n_step=200  
n_out=2
/ 

&field 
Ffld='gau'
omega=0.262961
sigma=2.,
t_mid=15.,
radiative='non'
fmax= 0.000001 0. 0.
/

&spectra 
start=0.  
tau= 2000.0 2000.0
dir_ft= 1. 0. 0. 
/

&sse 
dissipative='non'
/

! Here starts the tdplas part
!
&PROPAGATE  
propagation_type='ief'
interaction_type='pcm'
interaction_stride=1
interaction_init='non-scf'
debug_type='off' 
local_field='loc'
/

&MEDIUM
medium_type='nan'
medium_init='fro'
bem_type='diag'
bem_read_write='rea'
/

&SURFACE
input_surface='fil'
/

&EPS_FUNCTION
epsilon_omega='drl'
eps_0=1000.
eps_d=1. 
eps_A=0.110224
eps_gm=0.00757576
eps_w0=0.  
f_vel=0. 
/
