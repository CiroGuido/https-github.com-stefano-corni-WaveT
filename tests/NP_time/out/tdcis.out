 
 *****************************************************
 *****************************************************
 **                                                 **
 **                    WaveT                        **
 **     Evolution of molecular wavefunctions        **
 **   under external electromagnetic perturbations  **
 **                                                 **
 **                     by                          **
 **                Stefano Corni                    **
 **                Silvio Pipolo                    **
 **               Emanuele Coccia                   **
 **                 Gabriel Gil                     **
 **                Jacopo Fregoni                   **
 **                                                 **
 *****************************************************
 *****************************************************
 
 Number to append to dat file           1
 Number of CIS states to be read          15
 Number of CIS states to be used          10
 Molecular center of charge
  0.000000000000000E+000  0.000000000000000E+000  0.000000000000000E+000
 
 Time step (in au), number of steps, stride  0.200000000000000              200
           2
 
 Nanoparticle as external medium
 No restart
 Only |k> -> |0> relaxation
 
 Time shape of the perturbing fieldgau       
 time at the center of the pulse (au):   15.0000000000000     
 Width of the pulse (time au):   2.00000000000000     
 Frequency (au):  0.262961000000000     
 Maximum E field  1.000000000000000E-006  0.000000000000000E+000
  0.000000000000000E+000
 One pulse
 
 Excitation energies read from input, in Hartree
           1  0.240259571027011     
           2  0.245275850424543     
           3  0.245275850424543     
           4  0.249402796771377     
           5  0.249402796771377     
           6  0.266075954006985     
           7  0.291028728445992     
           8  0.291028728445992     
           9  0.305522652214393     
          10  0.332430489392951     
 
 Starting point for FT calculation  0.000000000000000E+000
 Artificial damping   2000.00000000000        2000.00000000000     
 Direction along which the field is oriented   1.00000000000000     
  0.000000000000000E+000  0.000000000000000E+000
 
 No dissipation
 
 Apparent charges are propagated
 Local field effects are included
 Medium charges do not follow the quantum jump
 Frequency of updating the interaction potential           1
 Use GS RF as coded in the ci_energy.inp values
 The coupling is of PCM type: pot \cdot chr
 Nanoparticle as external medium
 Medium polarization described by apparent charges
 Diagonal BEM furmulation
 This is a Classical BEM run
 This is full run reading matrix and boundary
 Medium polarization frozen to molecule in its GS
 Surface read from file cavity.inp
 Essential output written.
 DEBUG: Molecule - Medium interaction turned off
Done reading input & setting up the field, took     0.036s
 BEM surface and Matrixes S D have been read in
 BEM correctly initialized
 Done BEM diagonal
 Medium contribution to ground state free energy: -1.352428659314055E-003
 G_neq at t=0:  0.000000000000000E+000
 Initiated VV propagator
Done , total elapsed time     1.391s
